scripts/linters/custom_eslint_checks/rules/e2e-action.js

```diff
@@ -19,7 +19,7 @@
 
 'use strict';
 
-let ACTION_FUNCTIONS_TO_ENFORCE = ['click', 'sendKeys'];
+let ACTION_FUNCTIONS_TO_ENFORCE = ['click', 'sendKeys', 'setValue', 'keys'];
```

Comment:
What's the alternative for click? Also, is keys similar to setValue.

Reply:
click is same for both, and yes keys is somehow similar to setValue, the only differenence is setValue is faster than keys.

Reply:
keys cannot be used as the substitue as it can only be used with browser, like: browser.keys and then it input the value in the current active input element. So I removed keys from everywhere in the files.

Reply:
Thanks for explaining. Got it.